### DISCUSSION TITLE
Can it find local images?

### DISCUSSION THREAD

**Discussion by gerroon** [artiomn/markdown_articles_tool#27] [Q&A] [answered]
Hi

Can this tool find the missing images in the current vault and fix the local link? Ihave some broken links due to folder mismatch, I know the images existst, just that they d onot exists where some of these note expect them from.

**Comment by artiomn** [comment, marked as answer]
No, this is mainly for downloading remote images.
Your document must have valid links from which you can download images (even if it is a local file system).

But if you need it, functionality like this may be implemented, at least the tool can write warnings (containing links) when the image is not found.

**Reply by gerroon** [reply]
Thanks for the reply. That functionality will be very useful if it is implemented.

**Reply by artiomn** [reply]
Ok, hopefully I'll be back to developing the tool in two months.
Please, create issue about this. With a more detailed description of the requirement.
